assets/controller.yaml

```diff
@@ -262,7 +262,7 @@ spec:
           imagePullPolicy: IfNotPresent
           args:
             - --csi-address=/csi/csi.sock
-            - --probe-timeout=3s
+            - --probe-timeout=10s
```

Comment:
you need to do the same change in `assets/node.yaml` in the livenessprobe container

Reply:
I did, isn't?

Reply:
indeed, thanks